### DISCUSSION TITLE
Official Docker image for DNSCrypt-Proxy Client?

### DISCUSSION THREAD

**Discussion by YugiFanGX** [DNSCrypt/dnscrypt-proxy#2230] [Ideas]
Do you consider to provide official Docker support for DNSCrypt-Proxy Client?

**Comment by jedisct1** [comment]
https://hub.docker.com/r/klutchell/dnscrypt-proxy is well-maintained, and can be considered the official docker image.

**Reply by YugiFanGX** [reply]
So might there be an option to add this Image on the website under clients?

https://dnscrypt.info/implementations

**Reply by jedisct1** [reply]
This is not a different client. So maybe the dnscrypt-proxy documentation would be a better place?

**Reply by YugiFanGX** [reply]
Yes, I think so. Unfortunately I am not knowing about git. Would be nice if you can add this.
